internal/controller/replication.storage/status.go

```diff
@@ -46,26 +50,30 @@ func setPromotedCondition(conditions *[]metav1.Condition, observedGeneration int
 }
 
 // sets conditions when volume promotion was failed.
-func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGeneration int64) {
+func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGeneration int64, errMessage string, errFromCephCSI string) {
```

Comment:
```suggestion
func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGeneration int64, errMessage, errFromCephCSI string) {
```

As both types are `string`, there is no need to repeat it.

Reply:
```suggestion
func setFailedPromotionCondition(conditions *[]metav1.Condition, observedGeneration int64, msg, msgFromDriver string) {
```